### ISSUE TITLE
[Text Alignment] requirements.txt breaks text alignment

### ISSUE THREAD

**Issue by MrMondrian** [DDMAL/Rodan#894]
When running text alignment on a healthy test case in a fresh healthy gpu container, there is no problem parsing an image/transcript pair through ocr and text alignment (only the healthy case returns a good result, but all image/transcript pairs do not throw an error). However, if you run pip install -r requirements.txt, align_to_ocr.py is no longer able to do its job and throws the following error. This is for any image/transcript pair. @timothydereuse 

```
Traceback (most recent call last):
  File "test.py", line 14, in <module>
    result = align.process(raw_image, transcript, model_name)
  File "/code/Rodan/rodan/jobs/text_alignment/align_to_ocr.py", line 85, in process
    all_chars = perform_ocr.recognize_text_strips(image, cc_strips, ocr_model_name, verbose)
  File "/code/Rodan/rodan/jobs/text_alignment/perform_ocr.py", line 68, in recognize_text_strips
    from calamari_ocr.ocr.predict.predictor import MultiPredictor, PredictorParams
  File "/usr/local/lib/python3.7/dist-packages/calamari_ocr/ocr/predict/predictor.py", line 10, in <module>
    import tfaip.imports as tfaip_cls
  File "/usr/local/lib/python3.7/dist-packages/tfaip/imports/__init__.py", line 32, in <module>
    from tfaip.trainer.trainer import Trainer
  File "/usr/local/lib/python3.7/dist-packages/tfaip/trainer/trainer.py", line 38, in <module>
    from tfaip.trainer.callbacks.ema_callback import EMACallback
  File "/usr/local/lib/python3.7/dist-packages/tfaip/trainer/callbacks/ema_callback.py", line 24, in <module>
    from tfaip.trainer.optimizer.weights_moving_average import WeightsMovingAverage
  File "/usr/local/lib/python3.7/dist-packages/tfaip/trainer/optimizer/weights_moving_average.py", line 20, in <module>
    import tensorflow_addons.optimizers as addons_optimizer
  File "/usr/local/lib/python3.7/dist-packages/tensorflow_addons/__init__.py", line 22, in <module>
    from tensorflow_addons import callbacks
  File "/usr/local/lib/python3.7/dist-packages/tensorflow_addons/callbacks/__init__.py", line 17, in <module>
    from tensorflow_addons.callbacks.average_model_checkpoint import AverageModelCheckpoint
  File "/usr/local/lib/python3.7/dist-packages/tensorflow_addons/callbacks/average_model_checkpoint.py", line 18, in <module>
    from tensorflow_addons.optimizers.average_wrapper import AveragedOptimizerWrapper
  File "/usr/local/lib/python3.7/dist-packages/tensorflow_addons/optimizers/__init__.py", line 30, in <module>
    from tensorflow_addons.optimizers.discriminative_layer_training import (
  File "/usr/local/lib/python3.7/dist-packages/tensorflow_addons/optimizers/discriminative_layer_training.py", line 24, in <module>
    from keras import backend
  File "/usr/local/lib/python3.7/dist-packages/keras/__init__.py", line 25, in <module>
    from keras import models
  File "/usr/local/lib/python3.7/dist-packages/keras/models.py", line 19, in <module>
    from keras import backend
  File "/usr/local/lib/python3.7/dist-packages/keras/backend.py", line 37, in <module>
    from tensorflow.python.eager.context import get_config
ImportError: cannot import name 'get_config' from 'tensorflow.python.eager.context' (/usr/local/lib/python3.7/dist-packages/tensorflow/python/eager/context.py)
```

**Comment by sabrina0822** [comment]
My best guess is that this is related to each job in the GPU container having their own requirements.txt. We will be creating one large requirement.txt file for all the gpu jobs in the upcoming weeks so hopefully this will be resolved.

**Comment by sabrina0822** [comment]
Closing this because it should be fixed with #639
